v2.0/initialize-a-cluster.md

```diff
@@ -4,7 +4,7 @@ summary: Perform a one-time-only initialization of a CockroachDB cluster.
 toc: false
 ---
 
-<span class="version-tag">New in v1.1:</span> This page explains the `cockroach init` [command](cockroach-commands.html), which you use to perform a one-time initialization of a new multi-node cluster. For a full walk-through of the cluster startup and initialization process, see one of the [Manual Deployment](manual-deployment.html) tutorials.
+<span class="version-tag">New in v1.1:</span> This page explains the `cockroach init` [command](cockroach-commands.html), which you use to perform a one-time initialization of a new multi-node cluster. The target of the `cockroach init` command must appear in the `--join` flag of the [`cockroach start`](start-a-cluster.html) command which have been run on the other nodes in the cluster. For a full walk-through of the cluster startup and initialization process, see one of the [Manual Deployment](manual-deployment.html) tutorials.
```

Comment:
See comment for the v1.1 version of the page.

Reply:
Which comment are you referring to on the v1.1 version of the page? I'm not seeing any Github comments on that page.

Also I've gone ahead and fixed TC if that is what it was referring to.